app/assets/stylesheets/custom.css.scss

```diff
@@ -14,3 +14,60 @@
     color: #468847;
     text-align: left;
  }
+
+ input[type=text], select {
+     width: 20%;
+     padding: 12px 20px;
+     margin: 8px 0;
+     display: inline-block;
+     border: 1px solid #ccc;
+     border-radius: 4px;
+     box-sizing: border-box;
+     color: white;
+ }
+
+ input[type=submit] {
+     width: 20%;
+     background-color: #2E978A;
+     color: white;
+     padding: 14px 20px;
+     margin: 8px 0;
+     border: none;
+     border-radius: 4px;
+     cursor: pointer;
+ }
+
+ input[type=submit]:hover {
+     background-color: #45a049;
+ }
+
+ div {
+     border-radius: 5px;
+     background-color: #666;
+     padding: 20px;
+     color: white;
+ }
+
+
+a:link {
+    color: #258075;
+    text-decoration: none;
+}
+
+
+a:visited {
+    color: #258075;
+    text-decoration: none;
+}
+
+
+a:hover {
+    color: #34DCC7;
+    text-decoration: none;
+}
+
+
+a:active {
+    color: #315954;
+    text-decoration: none;
+}
```

Comment:
I am not sure about adding this css in this PR. There exits another story for that in the tracker. Feels out of scope.